### PR TITLE
Kube router network plugin

### PR DESCRIPTION
**Pull request by amon-ra**
After trying to install calico and contiv plugins with no luck, I have ported this and it is working like a charm.
It a simple l3 network plugin, I have installed using:
```
openshift_use_kube_router=True
openshift_use_openshift_sdn=False
os_sdn_network_plugin_name='cni'
openshift_node_sdn_mtu=1500
```

### CODE REVIEW

File: roles/kube_router/tasks/install.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    resource_kind: scc
+    resource_name: privileged
+    state: present
+
+- name: Kube-router | Set default selector for kube-system
+  run_once: true
+  command: >
+    {{ openshift_client_binary }}
+    --config={{ openshift.common.config_base }}/master/admin.kubeconfig
+    annotate  ns kube-system openshift.io/node-selector="" --overwrite
```
Comment (vrutkovs) [inline roles/kube_router/tasks/install.yml]: That would remove master nodeselector for api/controller/etcd too. Maybe its worth moving kube-router to a separate NS instead?

**Comment by amon-ra** [comment]
It is token from calico task. Ir is probably more correct

File: roles/kube_router/tasks/install.yml (hunk 1 above)
Comment (amon-ra) [reply roles/kube_router/tasks/install.yml]: I have updated the code to create a new namespace. ¿It is ok now?

**Comment by mtnbikenc** [comment]
Closing due to no recent activity. Please open a new PR if this change is still needed.
/close

**Pull request closed without merging**